app/components/candidate_interface/gcse_qualification_review_component.rb

```diff
@@ -115,7 +115,7 @@ def country_row
     end
 
     def naric_statment_row
-      return nil unless FeatureFlag.active?('international_gcses') && application_qualification.qualification_type == 'non_uk'
```

Comment:
As you said Toby's pulling this out 👍

Reply:
I've rebased and resolved all of these changes now.